content/es/governance.md

```diff
@@ -0,0 +1,137 @@
+# io.js Gobernabilidad del Proyecto
+
+## Comité Técnico
+
+El proyecto io.js está gobernado conjuntamente por un Comité Técnico (CT)
+que es responsable de la dirección a alto nivel del proyecto.
+
+El CT tiene autoridad final sobre este proyecto incluyendo:
+
+* Dirección técnica
+* Gobernabilidad del proyecto y su proceso (incluyendo esta política)
+* Política de contribución
+* Alojamiento del repositorio en GitHub
+* Lineamientos de conducta
+* Mantenimiento de la lista de Colaboradores adicionales
+
+Las invitaciones iniciales de membresía al CT fueron dados a personas
+que han sido contribuyentes activos a io.js, y que tienen experiencia
+significativa con la dirección del proyecto io.js. Se espera que la
+membresía evolucione con el tiempo conforme las necesidades del proyecto.
+
+Para la lista actual de los miembros del CT, consulte el proyecto
+[README.md](./README.md#current-project-team-members).
+
+## Colaboradores
+
+El repositorio [iojs/io.js](https://github.com/iojs/io.js) en Github es
+mantenido por el CT y Colaboradores adicionales que son añadidos por el
+CT de manera continua.
+
+Los individuos que hacen contribuciones significativas y valiosas se
+hacen Colaboradores y se les asigna privilegios de commit al proyecto.
+Estos individuos son identificados por el CT y su incorporación como
+Colaboradores se discute en la reunión semanal del CT.
+
+_Nota:_ Si usted realiza una contribución significativa y no se le asigna
+privilegios de commit, registre un asunto o contacte directamente a un
+miembro del CT y será tomado en cuenta en la proxima reunión del CT.
+
+Las Modificaciones de los contenidos del repositorio iojs/io.js se hacen
```

Comment:
* `Modificaciones` ---> `modificaciones` (the original content use upper case because it's the first word in the sentence)